test/test_mypy.py

```diff
@@ -313,6 +313,16 @@ def test_son_document_type(self) -> None:
     def test_son_document_type_runtime(self) -> None:
         client = MongoClient(document_class=SON[str, Any], connect=False)
 
+    @only_type_check
+    def test_create_index(self) -> None:
```

Comment:
Should this test run create_index instead of a transaction?

Reply:
Done